lifx-core/src/lib.rs

```diff
@@ -2380,10 +2408,12 @@ impl RawMessage {
     }
 
     /// Validates that this object was constructed correctly.  Panics if not.
-    pub fn validate(&self) {
-        self.frame.validate();
-        self.frame_addr.validate();
-        self.protocol_header.validate();
+    pub fn validate(&self) -> Result<(), Error> {
```

Comment:
Code looks good, but the comment on line 2410 needs a small update

Reply:
👍